### PR TITLE
add "group" and some tiny improve translation with zh-CN

### PR DESCRIPTION
**Pull request by maskbdsm**
hello, i finished a little zh-CN translation here ,please check it.

### CODE REVIEW

**Comment by johnbaku** [comment]
@maskbdsm oh damn... thanks!

Is there anyone that can review this please?

File: locales/zh.yml [hunk 1]
```diff
@@ -489,40 +489,40 @@ zh:
...
+      cc_curo: 信用卡
+      coingate: 比特币
       eps: EPS
-      giropay: giropay
+      giropay: GiroPay
       ideal: iDEAL
       interac: Interac e-Transfer
       mail: Physical Mail
-      paygarden: Major Gift Card
+      paygarden:礼品卡
```
Comment (Jenny-HT) [inline locales/zh.yml]:
```suggestion
      paygarden: 礼品卡
```

File: locales/zh.yml [hunk 2]
```diff
@@ -25,9 +25,9 @@ zh:
     step: 步骤
   edit_profile:
     sidebar_questions:
-      pics_and_vids_discoverability_a: Through FetLife’s search, your pics and vids are only findable by friends and followers. If you want other FetLifers to discover your kinkalicious pics and vids, you can update your privacy settings.
-      pics_and_vids_discoverability_link: Go to settings
-      pics_and_vids_discoverability_q: Pics and vid discoverability
+      pics_and_vids_discoverability_a: 当有人使用FetLife搜索时，你的怪癖照片和视频只会被朋友和粉丝看到。如果你想对所有人可见，请更新你的隐私设置。
```
Comment (Jenny-HT) [inline locales/zh.yml]:
```suggestion
      pics_and_vids_discoverability_a: 当有人使用FetLife搜索时，你的怪癖照片和视频只会被朋友和粉丝看到。如果你想对其他FetLife使用者可见，请更新你的隐私设置。
```

File: locales/zh.yml (hunk 1 above)
Comment (Jenny-HT) [reply locales/zh.yml]: It will fix the checks.

**Comment by nielsenramon** [comment]
Double-checked all translations myself with google translate, nice addition, thanks!

**Pull request merged**